src/components/Mission/ProfileMissions.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+export default function ProfileMissions() {
+  const { missionList } = useSelector((store) => store.missions);
+  const joinedMission = missionList.filter(
+    (mission) => mission.reserved === true,
+  );
+  if (joinedMission.length === 0) {
```

Comment:
- [ ] As required by project requirements, please kindly make use of React conditional rendering syntax in your Rockets and Missions.

### For Example:

```
const missionsDiv = missions.map((mission) => (
    <tr
      key={mission.mission_id}
      className="odd:bg-gray-200 divide-x divide-gray-300"
    >
      <td className="flex align-top pl-1">{mission.mission_name}</td>
      <td className="px-3 py-2">{mission.description}</td>
      <td>
        {mission.reserved && (
          <div className="status w-64 flex justify-around text-xs">
            <span className="badge bg-blue-400 text-white rounded-sm px-1 py-0.5 uppercase grid place-items-center">
              Active Member
            </span>
          </div>
        )}
        {!mission.reserved && (
          <div className="status w-64 flex justify-around text-xs">
            <span
              type="button"
              className="badge bg-gray-500 text-white rounded-sm uppercase grid place-items-center px-1 py-0.5"
            >
              not a member
            </span>
          </div>
        )}
      </td>
      <td>
        {mission.reserved && (
          <div className="w-64 flex justify-around text-xs">
            <button
              type="button"
              className="text-sm text-red-600 border-red-400 rounded-sm border-2 px-1 py-0.5"
              onClick={() => dispatch(leaveMission(mission.mission_id))}
            >
              {' '}
              leave mission
            </button>
          </div>
        )}
        {!mission.reserved && (
          <div className="w-64 flex justify-around text-xs">
            <button
              type="button"
              className="text-sm text-gray-600 border-gray-400 rounded-sm border-2 px-1 py-0.5"
              onClick={() => dispatch(joinMission(mission.mission_id))}
            >
              {' '}
              Join Mission
            </button>
          </div>
        )}
      </td>
    </tr>
  ));
```



The above is an example and may not apply directly to your code.